features/step_definitions/challege_gift_exchange_steps.rb

```diff
@@ -252,6 +252,24 @@
   step %{I should see "Work was successfully posted"}
 end
 
+When /^an assignment has been fulfilled in a gift exchange$/ do
+  step %{I am logged in as "mod1"}
+  step %{I have created the gift exchange "Awesome Gift Exchange"}
+  step %{I open signups for "Awesome Gift Exchange"}
+  step %{everyone has signed up for the gift exchange "Awesome Gift Exchange"}
+  step %{I have generated matches for "Awesome Gift Exchange"}
+  step %{I have sent assignments for "Awesome Gift Exchange"}
+  step %{I am logged in as "myname1"}
+  step %{I fulfill my assignment}
+end
+
+# we're not testing the process of rejection here, just that
+# it doesn't affect the completion status of the challenge assignment
+When /^I refuse my gift story "(.*?)"/ do |work|
```

Comment:
Ambiguous regexp literal. Parenthesize the method arguments if it's surely a regexp literal, or add a whitespace to the right of the `/` if it should be a division.